examples/webcrypto/README.md

```diff
@@ -0,0 +1,11 @@
+# k6 webcrypto examples
+
+In this directory, you will find examples of how to use the k6's `webcrypto` module in your k6 scripts.
+
+> [!IMPORTANT]
+> We do run the tests based on these examples, that's why we have a simple convention for each example:
+>
+> * It should do any `console.log`. Since we try to detect that output (log) contain `INFO` keyword.
```

Comment:
What does "do" mean in this: "It should do any `console.log`." I wonder if it's do or do not?

```suggestion
> * It should do any `console.log`. Since we try to detect that output (log) contains the `INFO` keyword.
```

Reply:
I could fix this after merge if needed